lib/html/pipeline/markdown_filter.rb

```diff
@@ -23,6 +23,7 @@ def initialize(text, context = nil, result = nil)
       def call
         options = [:GITHUB_PRE_LANG]
         options << :HARDBREAKS if context[:gfm] != false
+        options << :UNSAFE if context[:unsafe]
```

Comment:
I wasn't sure if it would be desired to be unsafe by default (like `gfm`) - so I went with "No", for now.